contents/teams/content/objectives.mdx

```diff
@@ -1,76 +1,125 @@
-* Newsletter ads and placements using new methods of tracking to validate.
+**What we'll ship**
+
+* A recurring monthly content review  
+* Dashboards and supporting materials
 
 **We'll know we're successful when**
-* Open rates above 40%  
-* We know what works and this becomes easy to maintain 
+
+* The team has clear visibility into how content is performing at both an overall and per-product level.  
+* Everyone can access and understand key metrics without needing to dig.  
+* Insights from reviews consistently shape our content planning and prioritization.
 
 </details>
 
 <details>
-  <summary>Long-term vision for newsletters</summary>
+  <summary>Newsletter</summary>
 
-**Owner:** <TeamMember name="Ian Vanagas" />, <TeamMember name="Lior Neu-ner" />, and <TeamMember name="Andy Vandervell" />
+**Owner:** Ian Vanagas
 
-**Motivation:** We want to maintain our existing newsletter schedule and quality standards while developing a stronger opinion on the future of newsletters at PostHog, audiences we want to reach, and what we're going to do to reach them in the short, medium and long-term.
+**Motivation:** We've scaled back our paid acquisition with the newsletter, so we're going to migrate to Beehiiv. We'll grow more through better integration and analytics. This will also help us tie the newsletter closer to the "marketing funnel".
 
 **What we'll ship:**
-* An awesome newsletter every two weeks
-* Newsletter offsite to workshop new ideas and long-term plans
-* Updated handbook pages on newsletter process + plans
-* Hiring plan for what's needed to execute our newsletter vision
 
-**We'll know we're successful when**
-* We're excited about our long-term plans for the newsletter
-* We ship a great newsletter every two weeks
-* We continue to see good engagement and feedback on newsletters  
-* No decline in important metrics (shares, organic subs, open rate etc.)
-* We successfully onboard a new newsletter writer in Q3 with clear goals to work on
+- A great newsletter every two weeks  
+- Audit of subscriber list to remove inactive subs  
+- Migrate to Beehiiv and set up better onboarding flow, analytics, referrals, better integrated in [posthog.com](http://posthog.com) and other nice features
+
+**We'll know we're successful when:**
+
+- Newsletter is growing faster than when we left Substack  
+- Newsletters getting more views each time
 
 </details>
 
 <details>
-  <summary>Shipping product engineer handbook</summary>
+  <summary>Better high-level messaging</summary>
 
-**Owner:** <TeamMember name="Andy Vandervell" />
+**Owner:** Ian Vanagas
 
-**Motivation:** We want to completely own the concept of the Product Engineer by creating its central text. We're 80% of the way there. Need to ship final 20%.
+**Motivation:** We’ve had a lot of changes in terms of positioning of the company lately and that has left a scattering of messaging across our site, content, and how we talk about ourselves. 
```

Comment:
Interested to follow this. Docs will largely follow/replicate how brand/marketing/editorial positions messaging